cmd/osd/main.go

```diff
@@ -431,6 +439,18 @@ func start(c *cli.Context) error {
 		if err != nil {
 			logrus.Fatalf("Failed to access TLS file information: %v", err)
 		}
+		if len(authenticators) > 0 {
+			// Initialize system token manager if an authenticator is setup
```

Comment:
probably want to do line 268 here.

either way, there must be a check for jwt-system-shared-secret not to be empty when oidc or share-secret has been set.